cmd/chaos-builder/main.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var (
+	log = zap.Logger(true)
+)
+
+type metadata struct {
+	Type string
+}
+
+func main() {
+	generatedCode := `// Copyright 2020 Chaos Mesh Authors.
```

Comment:
use `const ` to define this.

Reply:
I use `codeHeader` as a const to replace this. `generatedCode` is not const.